docs/concepts/what-is-filecoin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: What is Filecoin?
+---
+
+# What is Filecoin?
+
+**Filecoin is a peer-to-peer network that stores files on the internet, with built-in economic incentives to ensure files are stored reliably over time.**
+
+In Filecoin, users pay storage miners to store their files. Storage miners are computers responsible for storing files and proving they have stored the files correctly over time. Anyone who wants to store their files or get paid for storing other users’ files can join Filecoin. Available storage and pricing is not controlled by any single company. Instead, Filecoin facilitates open markets for storing and retrieving files that anyone can participate in.
+
+![Diagram showing a four-step process for storing data with Filecoin. First step: User pays miner to store their data. Second step: Miner commits publicly via Filecoin’s blockchain to storing the data. Third step: The network constantly verifies that miners are storing data correctly. Fourth step: User pays miner to retrieve data.](./images/what-is-filecoin.png)
+
+## For users
```

Comment:
Is there a name more specific than "users" that we should use for those who want to store files?

Reply:
I have replaced this with “User who wants to store a file” for now, but there is probably a better word out there.